contributor-growth/drafts/contributor-growth-framework/1-Motivating.md

```diff
@@ -0,0 +1,107 @@
+
+### Minimize Steps: Developer Environment Experience 
+
+The first setup experience is key. Make sure that the development requirements are clear. What languages does the project use? Where can a contributor find the tooling that they need to install to be successful during development? Leverage any tool that reduces steps or makes the entire process smoother.  
+
+Use scripts to install tooling or tests and be clear about what is being installed. Ideally, all your contributor needs is one command to set up the environment, maybe two to run tests. 
+
+If your project has external dependencies, Docker Compose is a great tool.  It allows you to put a database into a Docker Compose file, and let the contributor run docker-compose up. 
+And make sure everything is well documented in your docs and/or video tutorials. No matter how trivial a step is, if it's part of the process, document it. Remember that friction is dangerous, so always ask yourself "can I make this easier?"  
+
+Examples of tools that help minimize PR submission process
+Tool | What it does
+---|---
+Tilt | [Automate a lot of otherwise manual steps, significantly improving the contributor experience. ] >> needs proper description
+Docker compose | To put dependencies in a Docker Compose file
+docker | To run docker containers locally
+An IDE that they are comfortable with
+VSCode, IntelliJ IDEA, etc. | An editor that does syntax highlighting and catches errors will save you a ton of time. If you can provide workspace definition files that will configure their existing IDE installations around your codebase, that will drastically reduce friction
+  
+  
+# The Human Factor
+
+While interviewing maintainers from different projects, one thing became clear: The human factor is one of the most important aspects. Yes, contributors want to learn, they want to grow professionally, but they also want to be valued and belong.
+The more welcoming your community is, the more contributors you'll attract. Especially with new contributors, reach out, welcome them, start a quick conversation to make the first connection. You can ask them about their project, how long they've been using your project, what they are working on, you name it but connect with them on a human level.
+
+Building individual connections is more of a grassroots effort and gets increasingly difficult as the community grows. But for smaller projects, these 1:1 interactions are key, and you'll need to nurture relationships if you want people to volunteer their time. Then, master the art of "nerd sniping" by dangling a problem in front of contributors in the hopes of triggering their interest and getting them working on it. 
+
+## Community CRMs
+To foster an engaged community, keeping track of contributor activity, preferences, motivations, etc., is key. Community CRMs like Savannah or Orbit help maintainers keep a pulse on your community and provide real data versus mere gut feeling. 
```

Comment:
I have never heard of Community CRMs before! 👍 Do you use one for linkerd?

Reply:
Yes, we are still testing them. We want to use CMRS better and develop best practices. Happy to share once we know more.